OpenRA.Game/Sync.cs

```diff
@@ -27,6 +27,37 @@ public interface ISync { }
 
 	public static class Sync
 	{
+		public readonly struct Unsynced : IDisposable
```

Comment:
I'm a bit confused, where is this used?

Reply:
It was used in an earlier commit. I removed it now.

I used it around EndFrame. So that InputHandler could be (A)Sync and World unware. The downside was the sync would get calculated even when there are no input events - most of the time.